priv/tools/internal/select_gc_bucket.erl

```diff
@@ -149,23 +146,19 @@ handle_manifest({_UUID,
                 #state{threshold=Threshold} = _State)
   when ContentLength < Threshold ->
     {0, 0};
-handle_manifest({_UUID, ?MANIFEST{bkey=BKey={Bucket,_},
+handle_manifest({_UUID, ?MANIFEST{bkey=BKey={Bucket, Key},
```

Comment:
I'd recommend using `CSBucket` and `CSKey` as this will be confusing (originally I wrote this, sorry!)